plugin/input/http/http.go

```diff
@@ -215,6 +261,17 @@ func (p *Plugin) putSourceID(x pipeline.SourceID) {
 }
 
 func (p *Plugin) serve(w http.ResponseWriter, r *http.Request) {
+	ok := p.auth(r)
+	if !ok {
+		p.logger.Warn("auth failed",
```

Comment:
how about to  add metric for this case ?

Reply:
done